step-ca/provisioners.mdx

```diff
@@ -1077,6 +1108,21 @@ To get a client certificate for a hardware-bound private key on your YubiKey:
    }
    ```
 
+##### Device Attestation for TPM 2.0
+
+While `step-ca` supports the `tpm` attestation format of ACME `device-attest-01`,
+an end-to-end TPM-bound client certificate flow is not available at this time.
+
+A device needs to provide an attestation certificate in its response to an ACME `device-attest-01` challenge.
+Apple has an online attestation authority service for this.
+Yubico offers attestation certificates on their devices.
+
+But we have not seen a public or open source attestation authority that can issue attestation certificates for TPMs.
+TPMs also have privacy constraints that complexify attestation flows.
```

Comment:
Maybe add that requirements for the attestation protocol / validation might be use case specific too. Not impossible to implement, but loads of functionality and policy to think of.